extensions/wikia/CommunityPage/CommunityPageSpecialController.class.php

```diff
@@ -21,39 +21,43 @@ public function __construct() {
 	}
 
 	public function index() {
+		global $wgSitename, $wgWikiTopic;
+
 		$this->specialPage->setHeaders();
 		$this->getOutput()->setPageTitle( $this->msg( 'communitypage-title' )->text() );
 		$this->addAssets();
-
 		$this->wg->SuppressPageHeader = true;
 		$this->wg->SuppressFooter = true;
 
 		// queue i18n messages for export to JS
 		JSMessages::enqueuePackage( 'CommunityPageSpecial', JSMessages::EXTERNAL );
 
+		$cardModulesData = $this->getCardModulesData();
+		$defaultModulesLimit = max( 0, self::DEFAULT_MODULES_MAX - count( $cardModulesData[ 'modules' ] ) );
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "modules" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)